src/qt/sendcoinsdialog.cpp

```diff
@@ -273,28 +273,18 @@ void SendCoinsDialog::on_sendButton_clicked()
 
         QString recipientElement;
 
-        if (!rcp.paymentRequest.IsInitialized()) // normal payment
+
+        if(rcp.label.length() > 0) // label with address
         {
-            if(rcp.label.length() > 0) // label with address
-            {
-                QString displayedLabel = rcp.label;
-                if (rcp.label.length() > CHARACTERS_DISPLAY_LIMIT_IN_LABEL)  
-                {
-                    displayedLabel = displayedLabel.left(CHARACTERS_DISPLAY_LIMIT_IN_LABEL).append("..."); // limit the amount of characters displayed in label
-                }
-                recipientElement = tr("%1 to %2").arg(amount, GUIUtil::HtmlEscape(displayedLabel));
-                recipientElement.append(QString(" (%1)").arg(address));
-            }
-            else // just address
+            QString displayedLabel = rcp.label;
+            if (rcp.label.length() > CHARACTERS_DISPLAY_LIMIT_IN_LABEL)
             {
-                recipientElement = tr("%1 to %2").arg(amount, address);
+                displayedLabel = displayedLabel.left(CHARACTERS_DISPLAY_LIMIT_IN_LABEL).append("..."); // limit the amount of characters displayed in label
             }
+            recipientElement = tr("%1 to %2").arg(amount, GUIUtil::HtmlEscape(displayedLabel));
+            recipientElement.append(QString(" (%1)").arg(address));
```

Comment:
Is there a chance the length of the recipient element will exceed the display limit after the substitution?

Reply:
[We lowered the limit to fit the `...`](https://github.com/dogecoin/dogecoin/issues/3182#issuecomment-1328302280). Note that this is move-only (indentation) for this PR